app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -0,0 +1,112 @@
+    <string name="done">完成</string>
+    <string name="setup_complete_logo">设置完整的图标</string>
+    <string name="chats">会话</string>
+    <string name="settings">设置</string>
+    <string name="new_chat">创建会话</string>
+    <string name="select_platform">选择平台</string>
+    <string name="select_platform_description">选择您要在此会话中使用的平台，会话创建后无法更改，如果想更改这个设置，请重新创建一个会话</string>
+    <string name="enable_at_leat_one_platform">请先启用一个平台再创建会话</string>
+    <string name="chat_icon">会话的图标</string>
+    <string name="using_certain_platform">使用 %s</string>
+    <string name="confirm">确定</string>
+    <string name="cancel">取消</string>
+    <string name="go_back">返回</string>
+    <string name="arrow_icon">箭头图标</string>
+    <string name="on">开启</string>
+    <string name="off">关闭</string>
+    <string name="system_default">系统默认</string>
+    <string name="theme_settings">主题</string>
+    <string name="supported_soon">很快会支持</string>
+    <string name="theme_description">动态取色主题,暗黑模式</string>
+    <string name="openai_setting">OpenAI</string>
+    <string name="anthropic_setting">Anthropic</string>
+    <string name="google_setting">Google</string>
+    <string name="platform_setting_description">API密钥,模型,系统提示词</string>
+    <string name="enable_api">启用API</string>
+    <string name="api_key">API密钥</string>
+    <string name="api_model">选择模型</string>
+    <string name="dynamic_theme">动态取色</string>
+    <string name="dark_mode">暗黑模式</string>
+    <string name="key_icon">密钥的图标</string>
+    <string name="model_icon">模型的图标</string>
+    <string name="send">发送</string>
+    <string name="copy_text">复制内容</string>
+    <string name="retry">重新生成</string>
+    <string name="ask_a_question">问一个问题吧...</string>
+    <string name="some_platforms_disabled">一些API已被禁用</string>
+    <string name="token_not_set">没有设置Token</string>
+    <string name="token_set">Token设置为 %s*****</string>
+    <string name="close">关闭</string>
+    <string name="delete">删除</string>
+    <string name="chats_selected">选择 %d 个</string>
+    <string name="deleted_chats">已删除 %d 个</string>
+    <string name="delete_selected_chats">确定要删除选择的会话吗?</string>
+    <string name="this_operation_can_t_be_undone">此操作是不可逆的</string>
+    <string name="system_prompt">系统提示词</string>
+    <string name="temperature">温度</string>
+    <string name="top_p">核心采样(Top p)</string>
+    <string name="system_prompt_setting">系统提示词</string>
+    <string name="system_prompt_description">系统提示词是一段命令、指导和上下文信息，在模型与用户交流前提供给它</string>
+    <string name="temperature_icon">温度的图标</string>
+    <string name="top_p_icon">核心采样的图标</string>
+    <string name="system_prompt_icon">系统提示词的图标</string>
+    <string name="temperature_setting">温度</string>
+    <string name="temperature_setting_description">温度是一个注入到回答中的随机量，温度越低，答案越集中和确定;温度越高，答案越为灵活</string>
+    <string name="top_p_setting">核心采样</string>
+    <string name="top_p_setting_description">在核心采样中，我们按递减概率顺序计算每个后续令牌的所有选项的累积分布，并在达到该值指定的特定概率后将其截断。\n
+ \n
+ 大多数模型通常建议改变这个或温度，但不是两者都改变。</string>
```

Comment:
There is an extra newline inside this string. Is this intended? 

```suggestion
    <string name="top_p_setting_description">在核心采样中，我们按递减概率顺序计算每个后续令牌的所有选项的累积分布，并在达到该值指定的特定概率后将其截断。\n大多数模型通常建议改变这个或温度，但不是两者都改变。</string>
```
Also, please merge this string resource to one line like the above suggestion. Multiple lines inside strings.xml will not be actually shown as multiple lines in the app.